vendor/parser-extended/asttypes.mli

```diff
@@ -61,20 +61,9 @@ type 'a loc = 'a Location.loc = {
   loc : Location.t;
 }
 
-(* constant layouts are parsed as layout annotations, and also used
-   in the type checker as already-inferred (i.e. non-variable) layouts *)
-type const_layout =
-  | Any
-  | Value
-  | Void
-  | Immediate64
-  | Immediate
-  | Float64
-  | Word
-  | Bits32
-  | Bits64
-
-type layout_annotation = const_layout loc
+type layout = Layout of string [@@unboxed]
```

Comment:
Why change the name of this type? They're still constant layouts.

Reply:
done